db/schema.rb

```diff
@@ -12,79 +10,84 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_181_209_184_959) do
-  create_table 'notifications', force: :cascade do |t|
-    t.integer 'user_id'
-    t.string 'title'
-    t.string 'message'
-    t.boolean 'read'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.index ['user_id'], name: 'index_notifications_on_user_id'
+ActiveRecord::Schema.define(version: 2018_12_09_184959) do
+
+  create_table "notifications", force: :cascade do |t|
```

Comment:
I have a feeling that we somehow should commit ourselves to ' or "?! is this always generated differently?

Reply:
maybe we should discuss this with the other teams, so everybody can configure his rails the same way